lib/fastpush.py

```diff
@@ -0,0 +1,150 @@
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+import cgi
+import json
+import mxbuild
+import os
+from m2ee import logger
+import traceback
+import threading
+import buildpackutil
+import requests
+import subprocess
+
+
+ROOT_DIR = os.getcwd() + '/'
+MXBUILD_FOLDER = ROOT_DIR + 'mxbuild/'
+
+
+PROJECT_DIR = '.local/project'
+DEPLOYMENT_DIR = os.path.join(PROJECT_DIR, 'deployment')
+TMP_PROJECT_DIR = '.local/tmp_project'
```

Comment:
Might be good if we describe the workflow between MPK -> intermediate (tmp) directory -> deployment directory a bit more so we also know what, why and how in a few months from now?